src/borg/testsuite/compress_test.py

```diff
@@ -210,3 +210,32 @@ def test_specified_compression_level(c_type, c_name, c_levels):
 def test_invalid_compression_level(invalid_spec):
     with pytest.raises(argparse.ArgumentTypeError):
         CompressionSpec(invalid_spec)
+
+
+
+
+
+@pytest.mark.parametrize("data_length", [100, 500, 1000, 4000, 8192, 16384])
```

Comment:
maybe spread the params a little more: 0, 10, 100, 1000, .., MAX_DATA_SIZE.

you could also give (data_length, expected_padding) tuples instead of duplicating the padme implementation code to here.